src/utils.py

```diff
@@ -55,152 +55,37 @@ def get_body(url: str, random_seed: float = None) -> str:
-
-        output = ''.join(text_list)
-        output = encoding_mapper.map(output)
-    elif 'huffingtonpost.co.uk' in url:
-
-        # get content of article (as html tag)
-        article_content = soup.find(class_="entry__content-list js-entry-content")
-
-        # find the paragraph tags in the content element
-        content_list = article_content.find_all('p')
-
-        # get the text content
-        text_list = [item.string for item in content_list if item.string is not None]
-
-        output = ''.join(text_list)
-        output = encoding_mapper.map(output)
-    elif 'inews.co.uk' in url:
-
-        # get content of article (as html tag)
-        article_content = soup.find(class_="sc-hAnkBK hHEGbG")
-
-        # find the paragraph tags in the content element
-        content_list = article_content.find_all('p')
-
-        # get the text content
-        text_list = [item.string for item in content_list if item.string is not None]
-
-        output = ''.join(text_list)
-        output = encoding_mapper.map(output)
-    elif 'telegraph.co.uk' in url:
-
-        # get content of article (as html tag)
-        article_content = soup.find(class_="articleBodyText section")
-
-        # find the paragraph tags in the content element
-        content_list = article_content.find_all('p')
-
-        # get the text content
-        text_list = [item.string for item in content_list if item.string is not None]
-
-        output = ''.join(text_list)
-        output = encoding_mapper.map(output)
-    elif 'mirror.co.uk' in url:
-
-        # get content of article (as html tag)
-        article_content = soup.find(class_="article-page news sticky-header stick-sharebar")
-
-        # find the paragraph tags in the content element
-        content_list = article_content.find_all('p')
-
-        # get the text content
-        text_list = [item.string for item in content_list if item.string is not None]
-
-        output = ''.join(text_list)
-        output = encoding_mapper.map(output)
-    elif 'independent.co.uk' in url:
-
-        # get content of article (as html tag)
-        article_content = soup.find(class_="article-type-article amp-mode-mouse wrapped_by_ads takeover-loaded")
-
-        # find the paragraph tags in the content element
-        content_list = article_content.find_all('p')
-
-        # get the text content
-        text_list = [item.string for item in content_list if item.string is not None]
-
-        output = ''.join(text_list)
-        output = encoding_mapper.map(output)
-    else:
-        raise UrlException
+            # find the paragraph tags in the content element
+            content_list = article_content.find_all('p')
 
-    # body = soup.find('body')
-    # the_contents_of_body_without_body_tags = body.findChildren()
+            # get the text content
+            text_list = [item.string for item in content_list if item.string is not None]
 
-    return output
+            output = ''.join(text_list)
```

Comment:
Actually, can we just send out the paragraphs as a list of strings rather than joining them? I think it might be better if we print them out on a PDF as paragraphs as in the original article, or at least give the calling object he ability to do so...

Reply:
I see what you mean in principle. In terms of its readability by the end user, I think it would be much better if the visual format of the text matched the original news article, i.e. it should appear in the original paragraphs!

If I remove this .join method on this line though, it results in an error. Most recent traceback call is in the encoding_mappings.py file, as per below:

line 14, in map
    input_text = input_text.replace(key, value)
AttributeError: 'list' object has no attribute 'replace'